docker-compose.test.yml

```diff
@@ -0,0 +1,73 @@
+# Docker Compose for Testing with Neo4j
+version: '3.8'
+
+services:
+  postgres:
+    image: postgres:16
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: neynar_parquet_importer
+    restart: unless-stopped
+    ports:
+      - "25432:5432"  # Use different port for tests to avoid conflicts
+    volumes:
+      - ./data/postgres:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  neo4j:
+    image: neo4j:5.15
+    environment:
+      - NEO4J_AUTH=neo4j/testpassword
+      - NEO4J_PLUGINS=["apoc"]
+      - NEO4J_db_tx__log_rotation_retention__policy=false
+      - NEO4J_dbms_security_procedures_unrestricted=apoc.*
+    ports:
+      - "37474:7474"   # HTTP (test port)
+      - "37687:7687"   # Bolt (test port)
+    volumes:
+      - neo4j_data:/data
+      - neo4j_logs:/logs
+    healthcheck:
+      test: ["CMD", "cypher-shell", "-u", "neo4j", "-p", "testpassword", "RETURN 1"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  test-runner:
+    build: 
+      context: .
+      dockerfile: Dockerfile.test
+    depends_on:
+      postgres:
+        condition: service_started
+      neo4j:
+        condition: service_healthy
+    env_file:
+      - .env.test
```

Comment:
Please commit a verion of this file that can be public

Reply:
This is preventing these containerized tests from working out of the the box

Reply:
@topocount Pushed - should work out of the box now. Nothing seemed necessary to keep secret.